code/modules/mob/living/living.dm

```diff
@@ -792,3 +793,10 @@ default behaviour is:
 		update_icons()
 	return canmove
 
+/mob/living/proc/update_water() // Involves overlays for humans.  Maybe we'll get submerged sprites for borgs in the future?
+	return
```

Comment:
Deleted the original comment because I took two seconds to look and found the human specific version, feel free to disregard.